Xamarin.Forms.Core/Page.cs

```diff
@@ -209,9 +209,9 @@ public Task<bool> DisplayAlert(string title, string message, string accept, stri
 			return args.Result.Task;
 		}
 
-		public Task<string> DisplayPromptAsync(string title, string message, string accept = "OK", string cancel = "Cancel", string placeholder = null, int maxLength = -1, Keyboard keyboard = default(Keyboard))
+		public Task<string> DisplayPromptAsync(string title, string message, string accept = "OK", string cancel = "Cancel", string placeholder = null, int maxLength = -1, Keyboard keyboard = default(Keyboard), string initialValue = "")
```

Comment:
this is a breaking change, please provide an Obsolete overload with the old signature